receiver/mongodbreceiver/testdata/config.yaml

```diff
@@ -1,20 +1,6 @@
-receivers:
-  mongodb:
-    hosts:
-      - endpoint: localhost:27017
-    username: otel
-    password: $MONGO_PASSWORD
-    collection_interval: 60s
-
-processors:
-  nop:
-
-exporters:
-  nop:
-
-service:
-  pipelines:
-    metrics:
-      receivers: [mongodb]
-      processors: [nop]
-      exporters: [nop]
+mongodb:
```

Comment:
I've noted in #14567 that this and several other receivers I'm responsible for are missing a test that loads and validates this config. For the sake of avoiding merge conflicts, I'll add these tests after this set of PRs is merged.